src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -609,6 +610,59 @@ public void AssemblyResolve_RequestingAssembly()
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/22555", TargetFrameworkMonikers.Uap)]
```

Comment:
Looks like AssemblyResolve is not raised at all in CoreRT and Assembly.LoadFrom is not supported. Feels to me like both are by design to disallow changing assembly resolution. @jkotas, does that sound right?